app/models/miq_report/generator.rb

```diff
@@ -400,6 +400,11 @@ def build_create_results(options, taskid = nil)
 
     report = dup
     report.table = nil
+
+    if db_klass.respond_to?(:report_settings) && db_klass.report_settings.present?
```

Comment:
```suggestion
    unless db_klass.try(:report_settings).blank?
```

Not making a statement abou the the `unless` just about the `try`

Reply:
I changed also with little suggestion from rubocop.

yes, I believe that we can avoid such lines of code and we can refactor whole method, let's do it in other PR.

thanks